src/features/create/behaviors/CreateLabelBehavior.js

```diff
@@ -3,9 +3,10 @@ import { Behavior } from '../../../core/eventBus/Behavior';
 
 export class CreateLabelBehavior extends Behavior {
 
-    constructor (eventBus, commandStack, create, toolbox) {
+    constructor (eventBus, canvas, commandStack, create, toolbox) {
```

Comment:
We need to have core components, which are available from all injected classes without explicit injecting it. An issue could be nice for this.

Reply:
#156